src/pages/Login/WelcomeMessage.js

```diff
@@ -1,33 +1,37 @@
 import React from 'react';
 
 import { TARGET_API_LABELS } from '../../lib/api';
+import NextSteps from './NextSteps';
 
-const getWelcomeMessage = ({
-  isOnNewEnrollmentPage,
-  targetApi,
-  TARGET_API_LABELS,
-}) => {
-  if (isOnNewEnrollmentPage && targetApi === 'franceconnect')
-    return "Vous souhaitez intégrer le bouton d'identification FranceConnect à votre service en ligne, ";
-
-  if (isOnNewEnrollmentPage)
-    return `Vous souhaitez connecter votre service en ligne à « ${
-      TARGET_API_LABELS[targetApi]
-    } », `;
-
-  if (TARGET_API_LABELS[targetApi])
-    return `Vous souhaitez suivre le traitement d'une demande d'habilitation à « ${
-      TARGET_API_LABELS[targetApi]
-    } », `;
-
-  return "Vous souhaitez suivre le traitement d'une demande d'habilitation, ";
+const getWelcomeMessage = ({ isOnNewEnrollmentPage, targetApi }) => {
+  let message = '';
+  if (isOnNewEnrollmentPage) {
```

Comment:
as a fervent « [guard clause](https://refactoring.guru/replace-nested-conditional-with-guard-clauses) » supporter, I am not found of this rewriting. This new code is more prone to be extended into a nested conditional antipattern